packages/definitions-parser/package.json

```diff
@@ -29,7 +29,8 @@
   },
   "devDependencies": {
     "@types/fs-extra": "^9.0.8",
-    "@types/pacote": "^11.1.5"
+    "@types/pacote": "^11.1.5",
+    "ts-expose-internals": "latest"
```

Comment:
What API are we missing that this adds? Can we make it public in typescript if it's important?

Reply:
Now it’s just `ts.getNormalizedAbsolutePath`. I was doing some more granular stuff earlier but switched to public APIs. I can get rid of this.